pkg/minikube/constants/constants_linux.go

```diff
@@ -28,4 +28,4 @@ var SupportedVMDrivers = [...]string{
 }
 
 var DefaultMountDir = homedir.HomeDir()
-var DefaultMountEndpoint = "/hosthome"
+var DefaultMountEndpoint = "/mount-9p"
```

Comment:
Since these are the same across all drivers now (which makes sense), could you move it out of the constants_$OS file into the main one?

Also WDYT of making this "/minikube-host" or something.